tests/core/selection/manual/focusnestededitable.html

```diff
@@ -0,0 +1,19 @@
+<div id="square" style="width: 50px; height: 50px; background-color: yellow; margin: 10px 50% 10px 50%;"></div>
+
+<div id="editor" contenteditable="true">
+	<p>You can type here <strong contenteditable="false">This is uneditable part <em contenteditable="true">and you can type again inside this editable part.</em></strong></p>
+</div>
+
+<script>
+	var square = document.getElementById( 'square' ),
+		editor = CKEDITOR.replace( 'editor', {
+			extraPlugins: 'divarea',
```

Comment:
There is no need to load `divarea` as extra plugin. Just replace `wysiwygarea` with `divarea`.